prime-router/src/main/resources/db/migration/V40__digest_index.sql

```diff
@@ -0,0 +1,14 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+CREATE INDEX IF NOT EXISTS idx_report_file_blob_digest
```

Comment:
Any need for this to be set to create concurrently so it doesn't do anything exciting?

Reply:
This is the sql provided by Chris after testing.